packages/contracts-bedrock/src/dispute/weth/WETH98.sol

```diff
@@ -24,8 +24,12 @@ import { IWETH } from "src/dispute/interfaces/IWETH.sol";
 /// @title WETH98
 /// @notice WETH98 is a version of WETH9 upgraded for Solidity 0.8.x.
 contract WETH98 is IWETH {
-    string public constant name = "Wrapped Ether";
-    string public constant symbol = "WETH";
+    function name() external view virtual override returns (string memory) {
```

Comment:
See why you put these here given the old location of the constants, but can we place functions in their proper location in the file?

Reply:
i updated this